idea/idea-core/src/org/jetbrains/kotlin/idea/core/oldGenerateUtil.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.core
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.psi.KtClassOrObject
+import org.jetbrains.kotlin.psi.KtDeclaration
+
+@JvmOverloads
+fun <T : KtDeclaration> insertMembersAfter(
```

Comment:
Why we need  @JvmOverloads here? is it used from Java code

Reply:
Thanks! Looks like it's not needed anywhere. Let me remove it.